src/bin/keycloakify/BuildOptions.ts

```diff
@@ -40,8 +48,9 @@ export namespace BuildOptions {
         themeName: string;
         extraPages?: string[];
         extraThemeProperties?: string[];
-        //NOTE: Only for the pom.xml file, questionable utility...
```

Comment:
not 'only' for pom file, but also used when uploading the jar to a package registry, e.g. sonatype nexus or gitlab